testing/src/main/java/io/grpc/FrozenClock.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.grpc.internal;
+package io.grpc;
```

Comment:
We don't want this in this package. (I know that you must have it in this package for visibility.)